src/storage/mvcc/reader/mod.rs

```diff
@@ -84,7 +84,9 @@ impl<S: Snapshot> MvccReader<S> {
         self.key_only = key_only;
     }
 
+    /// Find data based on key and submission time ts
     pub fn load_data(&mut self, key: &Key, ts: u64) -> Result<Value> {
+        // usually storage index
```

Comment:
What does this mean?

Reply:
if key_only is true,there is a great possibility that the index is stored, so it returns Ok(vec![]);